aurora/src/components/contact/emailDisplayField.tsx

```diff
@@ -1,13 +1,21 @@
 import React from 'react';
 import styled from 'styled-components';
 
```

Comment:
Refacto the component into a more general component, which takes a text field as a prop